.github/workflows/firebasepod.yml

```diff
@@ -0,0 +1,24 @@
+name: firebasepod
+
```

Comment:
Nit: can you please add a comment here about what exactly it's testing?

Reply:
Done